mlir/include/mlir/Dialect/Tosa/Transforms/CMakeLists.txt

```
@@ -2,6 +2,10 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls -name TosaOpt)
 mlir_tablegen(PassesEnums.h.inc -gen-enum-decls)
 mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs)
+mlir_tablegen(Passes.capi.h.inc -gen-pass-capi-header --prefix Tosa)
+mlir_tablegen(Passes.capi.cpp.inc -gen-pass-capi-impl --prefix Tosa)
+mlir_tablegen(PassesEnums.capi.h.inc -gen-enum-decls -gen-pass-capi-header --prefix Tosa)
+mlir_tablegen(PassesEnums.capi.cpp.inc -gen-enum-defs -gen-pass-capi-impl --prefix Tosa)
```

Should I rename this to `TosaOpt`?

```suggestion
mlir_tablegen(Passes.capi.h.inc -gen-pass-capi-header --prefix TosaOpt)
mlir_tablegen(Passes.capi.cpp.inc -gen-pass-capi-impl --prefix TosaOpt)
mlir_tablegen(PassesEnums.capi.h.inc -gen-enum-decls -gen-pass-capi-header --prefix TosaOpt)
mlir_tablegen(PassesEnums.capi.cpp.inc -gen-enum-defs -gen-pass-capi-impl --prefix TosaOpt)
```

---

This seems like a weird singleton .. if anything I would `s/TosaOpt/Tosa` and update dependencies.

---

There are other mentions to `TosaOpt` in:
https://github.com/llvm/llvm-project/blob/79709498eaa57095a9888b882e94f6d15a244d4b/utils/bazel/llvm-project-overlay/mlir/BUILD.bazel#L11536

and 

https://github.com/llvm/llvm-project/blob/79709498eaa57095a9888b882e94f6d15a244d4b/mlir/include/mlir/InitAllPasses.h#L88